### PR TITLE
PMM-10244 Vacuum Monitoring Dashboard

### PR DESCRIPTION
**Pull request by gen1us2k**
[PMM-10244](https://jira.percona.com/browse/PMM-10244)

### CODE REVIEW

**Comment by gen1us2k** [comment]
I'm not sure I placed a dashboard on the correct folder for PoC of Vacuum Monitoring feature.

File: dashboards/PostgreSQL/PostgreSQL_Vacuum_Monitoring.json [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
```
Comment (anton-bystrov) [inline dashboards/PostgreSQL/PostgreSQL_Vacuum_Monitoring.json:21]: Here change to "false" please

**Review by anton-bystrov — CHANGES_REQUESTED** [review]
What about variables -

File: dashboards/PostgreSQL/PostgreSQL_Vacuum_Monitoring.json [hunk 2]
```diff
@@ -0,0 +1,1199 @@
...
+        },
+        "tooltip": {
+          "mode": "single"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PA58DA793C7250F1B"
```
Comment (solovevayaroslavna) [inline dashboards/PostgreSQL/PostgreSQL_Vacuum_Monitoring.json:1074]: @anton-bystrov can you check datasource? Don't we need to use Metrics here?
`"datasource": "Metrics"`

Comment (solovevayaroslavna) [reply dashboards/PostgreSQL/PostgreSQL_Vacuum_Monitoring.json:1074]: @anton-bystrov sorry, I see, you left a comment highter

**Review by JiriCtvrtka — APPROVED** [review]

**Review by anton-bystrov — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Pull request merged**